### DISCUSSION TITLE
Error while upgrading vaultwarden to 1.23.1

### DISCUSSION THREAD

**Discussion by paulo-dc** [dani-garcia/vaultwarden#2247] [Q&A]
Hello,

First, many thanks for this implementation. Really vaultwarden is really great !

But unfortunatly, I just faced a major issue, after upgrading to docker Image 1.23.1

I deployed vaultwarden in an Openshift environment.

After updating the release of the docker image and the pod creation, it crash loop at start, and in the log I got this error (in loop)

```
Running migration 20210830193501
Executing migration script 20210830193501/up.sql
[2022-01-20 10:16:32.061][vaultwarden::util][WARN] Can't connect to database, retrying: DieselMig.
[CAUSE] QueryError(
    DatabaseError(
        __Unknown,
        "Can't create table `<database_name>`.`emergency_access` (errno: 150 \"Foreign key constraint is incorrectly formed\")",
    ),
)
```

### Deployment environment

* vaultwarden version: 1.23.1
* Install method: Docker image
* MariaDB : 10.5.12

Please can you help me to solve this issue ?

BR

Paulo

**Comment by BlackDex** [comment]
Please search first before posting an issue.
https://github.com/dani-garcia/vaultwarden/wiki/Using-the-MariaDB-%28MySQL%29-Backend#foreign-key-errors-collation-and-charset

**Reply by paulo-dc** [reply]
Oups, I'm sorry,
I sincerly apologize .
I was in "panik mode" (fear to lose data), and I confess my searches were done badly under pressure.

I did a rollback, and everything is ok.

Many thanks for your answer, I'll read carrefully the link you provided, before newly upgrading.

Once again, I apologize, and next time I will keep calm and take more time to make good searches.
